app/apollo/resolvers/common.js

```diff
@@ -38,7 +38,30 @@ const validClusterAuth = async (me, queryName, context) => {
     }
     return;
   }
-}; 
+};
+
+var getAllowedChannels = async(me, orgId, action, field, context)=>{
```

Comment:
`field` is passed in to `getAllowedChannels` and `filterChannelsToAllowed` but it doesn't seem to be in use in those functions